### DISCUSSION TITLE
Element change mandatories in a change notice?

### DISCUSSION THREAD

**Discussion by fernarr** [OP-TED/eForms-SDK#593] [Q&A] [answered]
In the [change section documentation](https://docs.ted.europa.eu/eforms/latest/schema/change-notice.html#changeSection) says:

> There must exist at least one "efac:Change" element in the Change Notice.

I would lile to know if it is mandatory in a change notice exits at least one  `<efac:Change>` elements when the original notice is a TED XML notice.

Besides, I would like to know if this sentence produces a WARN or an ERROR. Would I send a change notice only with this `<efac:Changes>` element in any case?

```
               <efac:Changes>
                  <efbc:ChangedNoticeIdentifier>23402-2023</efbc:ChangedNoticeIdentifier>
                  <efac:ChangeReason>
                     <cbc:ReasonCode listName="change-corrig-justification">update-add</cbc:ReasonCode>
                  </efac:ChangeReason>
               </efac:Changes>
```

**Comment by YvesJo** [comment, marked as answer]
Hi,
The documentation should be updated to consider the case where a notice is cancelled and for which no Change node is needed. For other cases leading to text/value updates, such node shall exist in order to inform the reader about the updated notice section even if the original notice is a TEDXML one (note that the section identifiers to be used are the ones of the Change notice which is of type eForm).
Not all rules are currently in place and we are doing our best to have them in place as soon as possible. Currently no error would be sent in the absence of Change node.
